### PR TITLE
Fix Sphinx python docstring formatting error.

### LINKED ISSUE #13021: mxnet.ndarray.BatchNorm docs error
## Description
Sphinx throws an error when generating the docs for `mxnet.ndarray.BatchNorm`.

## Error
```
/home/ubuntu/incubator-mxnet/python/mxnet/ndarray/__init__.py:docstring of mxnet.ndarray.BatchNorm:48: WARNING: Literal block expected; none found.
```

### PR DESCRIPTION
**Pull request by frankfliu**
## Description ##
Fixes #13021

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [X] Changes are complete (i.e. I finished coding on this PR)
- [X] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [X] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [X] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by harshp8l** [comment]
@mxnet-label-bot add [pr-awaiting-review, doc]

File: src/operator/nn/batch_norm.cc [hunk 1]
```diff
@@ -567,10 +567,9 @@ axis to be the last item in the input shape.
 Both ``gamma`` and ``beta`` are learnable parameters. But if ``fix_gamma`` is true,
 then set ``gamma`` to 1 and its gradient to 0.
 
-Note::
-
-When fix_gamma is set to True, no sparse support is provided. If fix_gamma is set to False,
-the sparse tensors will fallback.
+.. Note::
+  When fix_gamma is set to True, no sparse support is provided. If fix_gamma is set to False,
```
Comment (aaronmarkham) [inline src/operator/nn/batch_norm.cc]: I think consistent use of the double ticks would be good here.
```suggestion
  When ``fix_gamma`` is set to True, no sparse support is provided. If ``fix_gamma is`` set to False,
```

File: src/operator/nn/batch_norm.cc [hunk 2]
```diff
@@ -567,10 +567,9 @@ axis to be the last item in the input shape.
 Both ``gamma`` and ``beta`` are learnable parameters. But if ``fix_gamma`` is true,
 then set ``gamma`` to 1 and its gradient to 0.
 
-Note::
-
-When fix_gamma is set to True, no sparse support is provided. If fix_gamma is set to False,
-the sparse tensors will fallback.
+.. Note::
+  When ``fix_gamma`` is set to True, no sparse support is provided. If ``fix_gamma is`` set to False,
```
Comment (aaronmarkham) [inline src/operator/nn/batch_norm.cc]:
```suggestion
  When ``fix_gamma`` is set to True, no sparse support is provided. If ``fix_gamma`` is set to False,
```

**Comment by kalyc** [comment]
@mxnet-label-bot add [pr-awaiting-merge]

**Comment by kalyc** [comment]
@mxnet-label-bot update [pr-awaiting-merge]

**Comment by aaronmarkham** [comment]
@mxnet-label-bot add [Doc]
